src/execution/index/art/art_key.cpp

```diff
@@ -33,6 +38,11 @@ void Key::CreateKey(ArenaAllocator &allocator, Key &key, string_t value) {
 	key.len = value.GetSize() + 1;
 	key.data = allocator.Allocate(key.len);
 	memcpy(key.data, value.GetDataUnsafe(), key.len - 1);
+
+	if (key.len > 1 && key.data[key.len - 2] == '\0') {
+		// FIXME: rethink this
+		throw NotImplementedException("Indexes cannot have contain null-terminated decoded BLOBs.");
```

Comment:
Haha I guess that makes sense.

Do people actually index blob columns?

Reply:
So far nobody has complained (outside of a fuzzer), so probably not :)